src/components/Calculator.js

```diff
@@ -0,0 +1,216 @@
+              onClick={() => handleClick('x')}
+            >
+              x
+            </Button>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="secondary"
+              className="w-100"
+              onClick={() => handleClick('4')}
+            >
+              4
+            </Button>
+          </Col>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="secondary"
+              className="w-100"
+              onClick={() => handleClick('5')}
+            >
+              5
+            </Button>
+          </Col>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="secondary"
+              className="w-100"
+              onClick={() => handleClick('6')}
+            >
+              6
+            </Button>
+          </Col>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="warning"
+              className="w-100"
+              onClick={() => handleClick('-')}
+            >
+              -
+            </Button>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="secondary"
+              className="w-100"
+              onClick={() => handleClick('1')}
+            >
+              1
+            </Button>
+          </Col>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="secondary"
+              className="w-100"
+              onClick={() => handleClick('2')}
+            >
+              2
+            </Button>
+          </Col>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="secondary"
+              className="w-100"
+              onClick={() => handleClick('3')}
+            >
+              3
+            </Button>
+          </Col>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="warning"
+              className="w-100"
+              onClick={() => handleClick('+')}
+            >
+              +
+            </Button>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs={6} className="p-0">
+            <Button
+              variant="secondary"
+              className="w-100"
+              onClick={() => handleClick('0')}
+            >
+              0
+            </Button>
+          </Col>
+          <Col xs={3} className="p-0">
+            <Button
+              variant="secondary"
+              className="w-100"
+              onClick={() => handleClick('.')}
+            >
+              .
+            </Button>
+          </Col>
+
+          <Col xs={3} className="p-0">
+            <Button
+              variant="warning"
+              className="w-100"
+              onClick={() => handleClick('=')}
+            >
+              =
+            </Button>
+          </Col>
+        </Row>
+      </>
+    );
+  }
```

Comment:
- [ ] I cannot run a few operations such as percent. Please work on it.

Reply:
If you look at the code provided by microverse, the operation is not a percentage, is the mod operation 

https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js

Reply:
Yes but it does work too Sir. I doesn't give any result.

Reply:
For me is working
![mod](https://user-images.githubusercontent.com/60273425/129237109-b1c2c80f-12ca-46d3-97ee-ff3a048d29e4.gif)

Can you share the steps you are doing so I can reproduce your error and fix it?

Reply:
Please request a review and let the code reviewer see your project